packages/block-library/src/code/index.js

```diff
@@ -22,11 +22,10 @@ export const settings = {
 	icon,
 	example: {
 		attributes: {
-			content: __( '// A "block" is the abstract term used' ) + '\n' +
-			__( '// to describe units of markup that,' ) + '\n' +
-			__( '// when composed together, form the' ) + '\n' +
-			__( '// content or layout of a page.' ) + '\n' +
-			__( 'registerBlockType( name, settings );' ),
+			content: sprintf(
+				__( '// A "block" is the abstract term used%1$s// to describe units of markup that%1$s// when composed together, form the%1$s// content or layout of a page.%1$sregisterBlockType( name, settings );' ),
+				'\n'
```

Comment:
I think it's fine to just place the line breaks within the string? Cc @mcsf

Reply:
Yes, please. A translators note could be added just in case, e.g.

```js
// translators: Preserve \n markers for line breaks
__( '// A "block" is the abstract term used\nto describe units of markup that,…
```

Reply:
@Shizumi see prior art in https://github.com/WordPress/gutenberg/pull/18372/files#diff-812531cdc96f05a28ab1fd2ced11451cR27

Reply:
Thank you, everyone.
Correspond.